test/parallel/test-tls-ticket.js

```diff
@@ -87,15 +87,15 @@ function start(callback) {
 }
 
 process.on('exit', function() {
-  assert.equal(ticketLog.length, serverLog.length);
-  for (var i = 0; i < naturalServers.length - 1; i++) {
+  assert.strictEqual(ticketLog.length, serverLog.length);
+  for (let i = 0; i < naturalServers.length - 1; i++) {
     assert.notEqual(serverLog[i], serverLog[i + 1]);
-    assert.equal(ticketLog[i], ticketLog[i + 1]);
+    assert.strictEqual(ticketLog[i], ticketLog[i + 1]);
 
     // 2nd connection should have different ticket
     assert.notEqual(ticketLog[i], ticketLog[i + naturalServers.length]);
```

Comment:
Ah, I'll go ahead and make changes tonight. I was initially just checking for `strictEqual`.